packages/extension/src/components/pages/AddNewTrustline/AddNewTrustline.tsx

```diff
@@ -278,14 +352,22 @@ export const AddNewTrustline: FC = () => {
           </>
         }
         transaction={TransactionStatus.Rejected}
+        {...(params.inAppCall ? { onClick: () => navigate(HOME_PATH) } : {})}
```

Comment:
Same comment than above:
Is there a use case when you would be inApp and not have some parameters? I would remove this line.

Reply:
Done